### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by jburel**
Add badge to readme and text to index page
cc @manics

### CODE REVIEW

**Comment by manics** [comment]
Thanks! Could you also add an explicit link to https://omero-guides.readthedocs.io/en/latest/upload/docs/import.html ? Although there's a badge it looks like a CI status, it's not obvious to people arriving at this repository they should click it to go to readthedocs:

image

**Comment by jburel** [comment]
If you click on the doc icon then you get to the upload page. I will check what are the options

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# This is a template useful to create guides
+# User guide showing how to upload data into OMERO
+
+
+[![Documentation Status](https://readthedocs.org/projects/omero-guide-upload/badge/?version=latest)](https://omero-guides.readthedocs.io/en/latest/upload/docs/import.html)
+
+Documentation at [Upload data](https://omero-guides.readthedocs.io/en/latest/upload/docs/index.html)
```
Comment (pwalczysko) [inline README.md]:
```suggestion
The documentation is deployed at [Upload data](https://omero-guides.readthedocs.io/en/latest/upload/docs/index.html)
```

File: docs/index.rst [hunk 2]
```diff
@@ -1,10 +1,18 @@
 Upload data
 ===========
 
+This section shows how to import data using the Desktop client, the Command Line Interface (CLI) and OMERO.dropbox.
+It also covers advanced import features used mainly in the context of the Image Data Resource (IDR).
```
Comment (pwalczysko) [inline docs/index.rst]:
```suggestion
It also covers advanced import features and post-import processing steps used mainly in the context of the Image Data Resource (IDR).
```

**Comment by pwalczysko** [comment]
Commited two suggestions. Otherwise ready to merge fmpov

**Pull request merged**